kubeflow/hack/create_kptfile.py

```diff
@@ -0,0 +1,114 @@
+"""This is a script created for the updating of the GCP Kpt packages.
+
+The purpose of this is to upgrade to using setters and substitutions
```

Comment:
nit: shall we add a comment to mention it should be in sync with kubeflow/manifest/hack/create_kptfile.py

Reply:
I think we can delete this in a subsequent PR; I just want to check it in so we have a record in source control but its only intended for one time use.

Reply:
SGTM